lago_images/images.py

```diff
@@ -185,6 +215,36 @@ def custom_build_action(self, *args, **kwargs):
             commands_file=self.spec.commands_file
         )
 
+        if self.spec.size == 0:
+            build_utils.resize_disk_image(
```

Comment:
If this flow is identical to the flow in `LibguestFSImage` it can be moved to a method in the `Image` class (or other function that you'll find appropriate).